test/e2e/singleNodeQuery.go

```diff
@@ -88,14 +90,16 @@ func main() {
 	}
 
 	// get multi nodes from redis for single node model and batch node model
-	redisIp := serviceInfo[0]
-	store := redis.NewRedisClient(redisIp, false)
+	remoteRedisIp := serviceInfo[0]
```

Comment:
Redis IP and port can also be used in local, not necessary to add remote to var name.